wdl-cli/src/inputs.rs

```diff
@@ -235,12 +240,15 @@ impl Inputs {
     }
 
     /// Attempts to coalesce a set of inputs into an [`Inputs`].
-    pub fn coalesce<T, V>(iter: T) -> Result<Self>
+    pub fn coalesce<T, V>(iter: T, name: Option<String>) -> Result<Self>
     where
```

Comment:
The docstring needs a description of `name`. I also notice that none of the tests within this repo use a `name` argument other than `None`. I realize that it is exercised with https://github.com/stjude-rust-labs/sprocket/pull/147 but could you please add some tests so we can have some local assurance and examples of the behavior?

Reply:
`coalesce` is essentially just `Inputs::new` and the `name` member is only accessed in `into_engine_inputs()` which isn't actually tested anywhere in `wdl-cli` at this point. I could add new tests that would use the new `libtest-mimic`? Don't think I'd finish that before I sign off for the day, but we can hold this off till tomorrow.